test/e2e/tests/send-eth.spec.js

```diff
@@ -298,9 +298,11 @@ describe('Send ETH from dapp using advanced gas controls', function () {
         dapp: true,
         fixtures: new FixtureBuilder()
           .withPermissionControllerConnectedToTestDapp()
-          .withNetworkControllerSupportEIP1559()
           .build(),
-        ganacheOptions,
+        ganacheOptions: {
+          ...ganacheOptions,
+          hardfork: 'london',
```

Comment:
Any idea why this change was needed?

Reply:
Sort of. The default hardfork we set in `withFixtures` is an old one where EIP-1559 isn't supported. Previously this test initialized the network controller state to set `networkDetails` to `{ EIPS: { 1559: true } }`, i.e., the controller already knew that the network supported EIP-1559, even if no provider was set yet. Although `getEIP1559Compatibility` is called when a provider is initialized, so this would eventually get changed in state to `{ EIPS: { 1559: false } }`, that happened after the `network` state was changed. It's possible that this test was running so fast that the MetaMask notification was spawned before `networkDetails` was updated. However, now when MetaMask loads, we guarantee that `networkDetails` is updated at the same time that `networkId` and `networkStatus` are updated, so when the notification window opens, `networkDetails` is ``{ EIPS: { 1559: false } }`. That's not what we want, obviously. Since we're using Ganache, we can just switch to a hardfork that supports EIP-1559 instead of manipulating state.